tensorbay/opendataset/UrbanObjectDetection/__init__.py

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env python3
+#
+# Copyright 2021 Graviti. Licensed under MIT License.
+#
+# pylint: disable=invalid-name
+
+"""Dataloader of UrbanObjectDetection."""
+
```

Comment:
need to add `UrbanObjectDetection` in `docs/source/reference/api/opendataset.rst`
also need to update `tensorbay/opendataset/__init__.py`